cmd/minikube/cmd/start.go

```diff
@@ -851,7 +851,7 @@ func validateUser(drvName string) {
 
 // memoryLimits returns the amount of memory allocated to the system and hypervisor, the return value is in MiB
 func memoryLimits(drvName string) (int, int, error) {
-	info, cpuErr, memErr, diskErr := machine.CachedHostInfo()
+	info, cpuErr, memErr, diskErr := machine.LocalHostInfo()
```

Comment:
any reason this was changed for all drivers ? previously it used a cache, this might add a small overhead on "docker driver" which needs invoking docker (that could cost 2-3 seconds on slow dockers)

Reply:
It _still_ uses the cache, I just renamed the function. If needed, we could keep the "CachedHostInfo" and give it a parameter ?

```go
// LocalHostInfo returns system information such as memory,CPU, DiskSize
func LocalHostInfo() (*HostInfo, error, error, error)
```

```go
// RemoteHostInfo returns system information such as memory,CPU, DiskSize
func RemoteHostInfo(r command.Runner) (*HostInfo, error, error, error)
```

This is how it is currently being used, but there is a timing difference (it needs SSH to be up first, to get the info)